modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step11_rgmb_het_hom.md

```diff
@@ -0,0 +1,128 @@
+- Use a unique [AssemblyMeshGenerator.md] block for each assembly with a unique geometrical configuration, region ID composition, and/or inventory of pin structures
+- background_region_ids is a 1-dimensional array containing region IDs for each axial layer of the background region. duct_region_ids is a 2-dimensional array containing regions IDs for the duct region. The first row of the array represents the 2D duct regions (from innermost to outermost duct region) for the bottom layer of the assembly. Each subsequent row assigns IDs on another axial level, from bottom to top.
+- While the mesh is still 2D during this step, the axially dependent region IDs are stored for later use during the extrusion step.
+
+## Homogeneous assembly structures using PinMeshGenerator
+
+In order to define homogeneous assembly structures to stitch into the core, we use [PinMeshGenerator.md] once again ([AssemblyMeshGenerator.md] is only used for structures that consist of lattices of pins).
+
+- +To define single assemblies directly with PinMeshGenerators for stitching with [CoreMeshGenerator.md]+, [PinMeshGenerator.md] is used with [!param](/Mesh/PinMeshGenerator/use_as_assembly) set to `true`.
+- In addition, [!param](/Mesh/PinMeshGenerator/homogenized) = `true` is used to indicate that this region is homogenized and [SimpleHexagonGenerator.md] should be called to discretize the assembly instead of [PolygonConcentricCircleMeshGenerator.md].
+
+!listing reactor_examples/rgmb_abtr/rgmb_abtr_het_mesh.i
+         id=tutorial04-rgmb_abtr-hom-assembly
+         caption=ABTR RGMB homogeneous assembly example.
+         block=Mesh/reflector_assembly
+
+Tips
+
+- For homogenized assemblies, each assembly has only 1 radial region, so each column in `region_ids` pertains to the region ID of each axial level of the homogenized assembly.
+- While the mesh is still 2D during this step, the axially dependent region IDs are stored for later use during the extrusion step.
+
+## Heterogeneous core using CoreMeshGenerator
+
+Now that all heterogeneous and homogeneous assemblies have been defined, they are placed into a lattice using [CoreMeshGenerator.md]. While [CoreMeshGenerator.md] still requires a perfect hexagonal pattern like [PatternedHexMeshGenerator.md], it automatically handles dummy assembly creation and deletion. The user need only provide a fake mesh input reference `dummy` (this object has not been actually created) and tell [CoreMeshGenerator.md] through the [!param](/Mesh/CoreMeshGenerator/dummy_assembly_name) parameter that the mesh input called `dummy` is a dummy assembly (empty space). The dummy assemblies will be created and deleted behind the scenes with no effort from the user.
+
+- Since we want to extrude the 2D core, we use the [!param](/Mesh/CoreMeshGenerator/extrude)=`true` parameter within [CoreMeshGenerator.md]. The step simultaneously extrudes the geometry and applies the regions IDs to all axial layers as defined in the [PinMeshGenerator.md] objects.
```

Comment:
it also processes the assembly MG ducts and background axial parameters too?

Reply:
Updated to include references to both PMG and AMG objects